spec/models/evaluation_form_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe EvaluationForm, type: :model do
+  describe 'validations' do
```

Comment:
we can validate that an "empty" EvaluationForm that doesn't have any data is still saved and a test for the uniqueness constraint for challenge_id / phase.